internal/pkg/plugininstaller/jenkins/validate.go

```diff
@@ -73,7 +96,7 @@ func buildAdminToken(userName string) (*jenkins.BasicAuth, error) {
 			Password: jenkinsPassword,
 		}, nil
 	}
-	// 2. if not set, get user and password from secret
+	// 2. if not set, get User and password from secret
```

Comment:
why is `User` and not `user`

Reply:
It's "user" here, changed by ide refactor